test/e2e/specs/specs-playwright/wp-seo__preview-spec.js

```diff
@@ -13,7 +13,7 @@ describe( DataHelper.createSuiteTitle( 'SEO Preview Page' ), function () {
 
 	it( 'Navigate to Tools > Marketing page', async function () {
 		const sidebarComponent = await SidebarComponent.Expect( this.page );
-		await sidebarComponent.gotoMenu( { item: 'Tools' } );
+		await sidebarComponent.gotoMenu( { item: 'Tools', subitem: 'Marketing' } );
```

Comment:
The `Marketing` subitem is selected by default if one opens `Tools`, but this is good practice to enforce anyway since that behavior can change at any moment.